integration-tests/test.sh

```diff
@@ -1,68 +1,95 @@
 #!/usr/bin/env bash
+# shellcheck source=../ci/lib/stdlib.bash
+source "$(dirname "${BASH_SOURCE[0]}")/../ci/lib/stdlib.bash" || exit 67
 
-set -xeu
+os="${1:?"Need OS as 1st arg. e.g. alpine arch centos7 precise wheezy"}"
+arch="${2:?"Need arch as 2nd arg. e.g. amd64 386"}"
 
-os=$1
-arch=$2
 vars_inline="{inline: bar, overwrite: bar}"
 
 seccomp_opts() {
   local docker_ver minor_ver
   docker_ver=$(docker version -f '{{.Client.Version}}')
+  log_info "docker version: ${docker_ver}"
   minor_ver=$(cut -d'.' -f2 <<<$docker_ver)
   if ((minor_ver>=10)); then
```

Comment:
```suggestion
  if ((10#$minor_ver>=10)); then
```

Should fix the whole: value too great for base error
https://stackoverflow.com/questions/12821715/convert-string-into-integer-in-bash-script-leading-zero-number-error/12821845#12821845

Reply:
🤦 thanks!